github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.42.yaml

```
@@ -59,6 +59,35 @@ presubmits:
               - "/bin/sh"
               - "-c"
               - "automation/check-patch.e2e-workflow-k8s.sh"
+    - name: pull-cluster-network-addons-operator-unit-test
+      branches:
+        - release-0.42
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      max_concurrency: 6
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: kubevirtci/bootstrap:v20201119-a5880e0
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "automation/check-patch.unit-test.sh"
```

docker in docker requires privileged mode.
```suggestion
              - "automation/check-patch.unit-test.sh"
            securityContext:
              privileged: true
```

---

I think the container is already configured to privileged mode [here](https://github.com/kubevirt/project-infra/pull/811/files#diff-61d6d3234278918d4dbac6e78ff300a63d0c6a493bdd75b1797d39a6ef790cfcR82). do I need to add the privilege configuration somewhere else?

---

Indeed that's perfect, sorry, I missed it